src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/AnalyzerReference.xaml

```diff
@@ -7,7 +7,8 @@
     Description="Analyzer reference properties"
     xmlns="http://schemas.microsoft.com/build/2009/properties">
     <Rule.DataSource>
-        <DataSource Persistence="ProjectFile" ItemType="Analyzer" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext" />
+        <DataSource Persistence="ProjectFile" ItemType="Analyzer" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext"
+                    SourceType="TargetResults" MSBuildTarget="CollectAnalyzersDesignTime"/>
```

Comment:
We've said that we'll do loc as a separate pass. There are other changes that need to be loc'ed as well. So we can do them all in one go.